components/ProductMatrix.vue

```diff
@@ -1,16 +1,20 @@
 <template>
   <div>
-    <div v-if="error">
-      {{ error }}
-    </div>
-    <div class="product-matrix">
-      <div v-for="card in cards" :key="card.sortKey">
-        <component :is="components[card.type]" :card="card" />
+    <div ref="matrix" class="product-matrix">
+      <div v-for="(card, index) in cards" :key="card.sortKey">
+        <component
+          :is="components[card.type]"
+          v-if="index === 0"
+          ref="card"
+          :card="card"
+        />
+        <component :is="components[card.type]" v-else :card="card" />
       </div>
+      <div v-for="n in fillersNeeded" :key="n" class="card fill-last-row" />
```

Comment:
@benjick Did you decide to use this solution until #12 is done?

Reply:
Updated 👍